modules/serving-HA-components.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-config-HA.adoc
+
+[id="serving-HA-components_{context}"]
+= Knative Serving HA components
+
+Active/passive high availability (HA) is available by default for the `autoscaler-hpa`, `controller`, `activator` , `kourier-control`, and `kourier-gateway` components.
+
+You can disable HA for these components by changing the configuration of the `KnativeServing.spec.highAvailability` field.
+
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field, as this will cause your deployment to become unsupported.
+====
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed the {ServerlessOperatorName} and Knative Serving.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the *Catalog* → *OperatorHub* → *Installed Operators* page.
+. Select the *{ServerlessOperatorName}*.
+. Click *Knative Serving* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Serving* tab.
++
+image::serverless-installed-operator.png[Installed Operators page]
+. Click on `knative-serving` in the *Knative Serving* tab.
+. Click on `YAML` in the *knative-serving* page.
++
+image::serving-YAML-HA.png[Knative Serving YAML]
+. In the `spec` section of the YAML, you will see the `high-availability` field configured with `2` replicas:
```

Comment:
Unfortunately. This image has so much text that it is not readable in the preview. Maybe we could cut off the right side (let's say from the middle of the image) and zoom in?

Reply:
Tried removing the the outer part and just showing the YAML - we need to make sure the image still makes sense and isn't cutting off in a weird way. Unfortunately it's an issue with most images in OCP docs that the text can look very small.

Reply:
I would just say to consider whether these screenshots are truly necessary or not. We don't use screenshots a lot and have plenty of GUI procedures in the docs.

Reply:
Also, "you will see the `high-availability` field" -> "the `high-availability` field is configured "